qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -299,6 +301,29 @@ def test_query_client_filter(self):
         log.debug("metrics={0}".format(metrics))
         self.assertTrue(valid)
 
+    def test_query_client_ip_filter(self):
```

Comment:
these cases are covered - see `test_query_client_filter` and `test_query_mds_and_client_filter`

Reply:
@vshankar  thought to add separate test for ```--client _ip```